stubs/gdb/gdb/__init__.pyi

```diff
@@ -8,7 +8,12 @@ from contextlib import AbstractContextManager
 from typing import Protocol, overload
 from typing_extensions import TypeAlias
 
+# The following submodules are automatically imported
+import gdb.events as events
```

Comment:
This still isn't *quite* right; the name following the `as` needs to be identical to the name before the `as` in order for it to be considered a public re-export. Something like this should work:

```suggestion
from . import events as events
```

Not sure why mypy isn't emitting an error for you locally (might be something to do with config settings or something), but this is how it's *meant* to work :)

Reply:
Ah right, I should have read the PEP more carefully. This should be fixed now, thank you.